controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
> A test should capture the expected behavior in different scenarios.


@aslakknutsen I will seperate the test cases and handle one scenario per test case. There are following scenarios:
[1] Reorder item between two items:
            [a] using above
            [b] using below
[2] Reorder item to top
[3] Reorder item to bottom
[4[ Reorder multiple workitems